### PR TITLE
Add cellfinder_core specific logger

### PR DESCRIPTION
**Pull request by dstansby**
This adds a logger that is specific to `cellfinder_core`, instead of using the root Python logger. This allows one to customize the `cellfinder-core` logger independently of other log messages, e.g. to show DEBUG messages just from cellfinder-core but not other packages:

```python
import logging
from cellfinder_core import logger

setLevel(logging.DEBUG)
```

### CODE REVIEW

File: src/cellfinder_core/__init__.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
 __version__ = "0.3.0"
 __author__ = "Adam Tyson, Christian Niedworok, Charly Rousseau"
 __license__ = "BSD-3-Clause"
+
+import logging
```
Comment (paddyroddy) [inline src/cellfinder_core/__init__.py:5]: I don't know what the "standard" way of doing it in `__init__.py`. Should the import be here or at the top?

File: src/cellfinder_core/main.py [hunk 2]
```diff
@@ -105,9 +106,9 @@ def main(
             callback=classify_callback,
         )
     else:
-        logging.info("No candidates, skipping classification")
+        logger.info("No candidates, skipping classification")
     return points
-    # logging.info("Saving classified cells")
+    # logger.info("Saving classified cells")
     # save_cells(points, classified_points_path)
```
Comment (paddyroddy) [inline src/cellfinder_core/main.py:112]: Do these comments still need to be here?

**Review by paddyroddy — APPROVED** [review]

File: src/cellfinder_core/__init__.py (hunk 1 above)
Comment (dstansby) [reply src/cellfinder_core/__init__.py:5]: 🤷 I'm not sure but isort isn't complaining, so I guess this is fine?

File: src/cellfinder_core/main.py (hunk 2 above)
Comment (dstansby) [reply src/cellfinder_core/main.py:112]: I'm not sure so I thought it best to leave them

**Pull request merged**